lib/stat_tracker.rb

```diff
@@ -33,24 +33,26 @@ def percentage_home_wins
     home_wins = @game_teams.count do |game|
       game.result == "WIN" && game.hoa == "home"
     end
-
-    (home_wins / self.total_games).round(2)*100
+    (home_wins / self.total_games).round(2).to_f
   end
   
   def percentage_visitor_wins
     visitor_wins = @game_teams.count do |game|
       game.result == "WIN" && game.hoa == "away"
     end
-
-    (visitor_wins / self.total_games).round(2)*100
+    (visitor_wins / self.total_games).round(2).to_f
   end
 
   def percentage_ties
     ties = @game_teams.count do |game|
       game.result == "TIE" && game.hoa == "away"
     end
+    (ties / self.total_games).round(2).to_f
+  end
 
-    (ties / self.total_games).round(2)*100
+  def average_goals_per_game
+    total_goals = @games.map { |game| game.away_goals + game.home_goals }.sum.to_f
+    (total_goals / @games.count).to_f.round(2)
```

Comment:
Nice utilization of a previously completed method. Keeps our production branch clean and adhers to DRY principle.